### DISCUSSION TITLE
Question 91:  Where to attach field notes? [ZK - Action]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#685] [Q&A] [answered]
Description: This is a follow up to questions 54 and 85. We have a PDF of field notes that correspond to these photos. Field notes are different for each group of photos, so where should I attach the MM record of the field notes? To each individual MM record it describes, to the Group record of photos, somewhere else? I didn’t see this on the docs linked in the previous questions. (see Events irn 2660; Groups irn 15735). Additionally, if I want to restrict Groups irn 15735 to Action and Anthro only, how would I do that? Is that just done with the departments on the Security tab?

**Comment by rondlg** [comment, marked as answer]
Response: 
- Attach field notes MM to main Event for start, or if have dates/etc, attach them to sub-Event for specific field-trip [with corresponding group of media]

- If need to restrict media to Action & Anthro only: record-level security.
 [See How do I update record level security on a group of records](https://github.com/fieldmuseum/EMu-Documentation/discussions/359)

**Reply by zkachian** [reply]
To clarify, should the MM record of the field notes be attached to the sub-Event as Associated Record (event 2 tab) even though it is just a single record and not necessarily a "group"? Or should it be attached somewhere else? I could include the field notes record in a group of images it corresponds to, but I don't know if it would get lost in there? See Event irn 9018 for example

**Comment by rondlg** [comment]
In the first instance - attach the fieldnotes mm record to the sub-event on the multimedia tab.

**Reply by zkachian** [reply]
Ok, that makes sense, thanks!
